lib/offsite_payments/integrations/sage_pay_form.rb

```diff
@@ -68,6 +68,13 @@ def cipher(action, key, payload)
 
       class Helper < OffsitePayments::Helper
         include Encryption
+        attr_reader :timestamp
+
+        def initialize(order, account, options = {})
+          super
+          @timestamp   = Time.now.strftime('%Y%m%d%H%M%S')
+          add_field 'VendorTxCode', "#{order}#{@timestamp.to_i}"
```

Comment:
Don't really need `to_i` on `timestamp` here because it's being interpolated back to a string.